pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -32,6 +32,8 @@ import (
 	drapbv1 "k8s.io/kubelet/pkg/apis/dra/v1alpha1"
 )
 
+const PluginTimeout = 10 * time.Second
```

Comment:
Can you call this PluginClient timeout

also, I remember there were two places where this constant should be used, not just one

Reply:
thank you for the review! done